kernelci/cli.py

```diff
@@ -229,6 +229,11 @@ class Args(object):
         'help': "Relative path where build artifacts are published",
     }
 
+    debos_path = {
+        'name': '--debos-path',
```

Comment:
This option name is a bit misleading as it looks like the path to the `debos` command.  Maybe something like `--data-path` would be better, also it would not be specific to debos.

Reply:
okay changed it.